agent/jvm/src/test/java/org/jolokia/jvmagent/security/KeyStoreUtilTest.java

```diff
@@ -59,6 +59,33 @@ public void testTrustStore() throws KeyStoreException, CertificateException, NoS
         assertEquals(key.getAlgorithm(),"RSA");
     }
 
+    @Test
+    public void testTrustStoreWithMultipeEntries() throws KeyStoreException, CertificateException, NoSuchAlgorithmException, IOException {
+        File caPem = getTempFile("ca/cert-multi.pem");
+        KeyStore keystore = createKeyStore();
+
+        KeyStoreUtil.updateWithCaPem(keystore, caPem);
+
+        Enumeration<String> aliases = keystore.aliases();
+        String alias = aliases.nextElement();
+        assertTrue(aliases.hasMoreElements());
+        assertTrue(alias.contains("another.test.jolokia.org"));
+        X509Certificate cert = (X509Certificate) keystore.getCertificate(alias);
+        cert.checkValidity();
```

Comment:
This seems unnecessary and rather incorrect. The validity check `java.security.cert.X509Certificate#checkValidity()` throws `java.security.cert.CertificateExpiredException` so this test is bound to start failing in year 2025.

Reply:
Agreed, we should probably only check whether the keystore can be parsed and has multiple elements.

Reply:
This's also true for former tests. Let me know if I should address that as well or a separate issue/PR is required.